lib/note-editor.jsx

```diff
@@ -232,6 +232,7 @@ export class NoteEditor extends Component {
               dangerouslySetInnerHTML={{
                 __html: renderNoteToHtml(content),
               }}
+              onClick={event => event.preventDefault()}
```

Comment:
what's the purpose here? will this not prevent clicking on links?

Reply:
Yeah I guess I double-fixed the issue. I can probably just remove this, but I was trying to disable clicks on the print div.